--- com/gliwka/hyperscan/jni/match_event_handler.java    2021-01-31 21:02:42.826119574 +0100
+++ com/gliwka/hyperscan/jni/match_event_handler.java    2021-01-31 21:02:42.826119574 +0100
@@ -69,6 +69,7 @@
  *      immediately return with \ref HS_SCAN_TERMINATED.
  */
 @Properties(inherit = com.gliwka.hyperscan.jni.JavaCppPreset.class)
+@Allocator(max = 256)
 public class match_event_handler extends FunctionPointer {
     static { Loader.load(); }
     /** Pointer cast constructor. Invokes {@link Pointer#Pointer(Pointer)}. */
